auacm/angular/src/app/blog-list/blog-list.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { Converter } from 'showdown';
+
+import { BlogService } from '../blog.service';
+import { AuthService } from '../auth.service';
+import { LimitWordsPipe }from '../pipes/limit-words.pipe';
+
+import { BlogPost } from '../models/blog';
+import { UserData } from '../models/user';
+
+@Component({
+  templateUrl: './blog-list.component.html',
+  styleUrls: ['./blog-list.component.css']
+})
+export class BlogListComponent implements OnInit {
+
+  converter: Converter = new Converter();
+
+  blogPosts: BlogPost[] = [];
+
+  user: UserData;
+
+  hoverId: number = -1;
```

Comment:
This hoverId value of -1 seems like a bit of a magic number. (It is also used auacm/angular/src/app/blog-list/blog-list.component.html, line 7). Can you define a constant and use that instead?